times-table.js

```diff
@@ -0,0 +1,25 @@
+var pry = require('pryjs');
+
+function timesTable(number){
+  string = "";
+  firstArray = createArray(number);
+  secondArray = createArray(number);
+  for (var i = 0; i < number; i++) {
+    for (var j = 0; j < number ; j++) {
+      string += (firstArray[i] * secondArray[j]);
+      string += " ";
+    }
+    string += '/n';
+  }
+  return string;
+}
+
+function createArray(number){
```

Comment:
This is an extra step you don't need when iterating you should just be able to just `i` and `j` since the numbers are increasing by 1 and these equal what you are setting the array values to anyway